helm-chart/jupyterhub-ssh/values.yaml

```diff
@@ -23,6 +23,9 @@ fullnameOverride: "jupyterhub"
 ssh:
   enabled: true
   replicaCount: 1
+  # Set timeout period for ssh connectoin. May need to set higher if your server
+  # a long time to start. 
+  timeout: 30
```

Comment:
I think we should use a more general configuration system than we currently do before it's too late and cause lots of breaking changes.

```suggestion

  config:
    JupyterHubSSH:
      start_timeout: 30
```

This config can then be loaded like [here](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/8be9bc14597d9b17555e2c6bd85829a2cd604d44/jupyterhub/files/hub/jupyterhub_config.py#L431-L441) in the JupyterHub Helm chart that does the same kind of thing.

```python
# load generic configuration
# As an example,
# c.JupyterHubSSH.start_timeout = 30
# could be set via
# hub.config.JupyterHubSSH.start_timeout = 30
# thanks to this.
for app, cfg in config["ssh"]["config"].items():
    c[app].update(cfg)
```